common/apm/src/metrics.rs

```diff
@@ -54,3 +54,18 @@ pub fn all_metrics() -> ProtocolResult<Vec<u8>> {
 
     Ok(encoded_metrics)
 }
+
+#[cfg(test)]
+mod tests {
+    use std::time::{Duration, Instant};
+    use super::duration_to_sec;
+
+    #[test]
+    fn test_duration_to_sec() {
+        let inst = Instant::now();
+        let d = Duration::from_nanos(10000000010);
+        let sec = duration_to_sec(d);
+        let sec2 = duration_to_sec(inst.elapsed());
+        println!("sec {:?} sec2 {:?}", sec, sec2);
```

Comment:
`assert_eq` would be better, try to more cases like _2s_, _1.1s_, _0.11s_, _0.000000003s_